pulp_ansible/app/models.py

```diff
@@ -211,6 +212,29 @@ class Meta:
         unique_together = ("pubkey_fingerprint", "signed_collection")
 
 
+class CollectionDownloadLog(BaseModel):
+    """
+    A download log for collection versions by user, IP and org_id
+    """
+
+    # Data Fields
+    collection_version = models.ForeignKey(
+        CollectionVersion, on_delete=models.CASCADE, related_name="download_logs"
+    )
+    user = models.ForeignKey(
+        get_user_model(), on_delete=models.CASCADE, null=True, related_name="collection_download_logs"
+    )
+    ip = models.GenericIPAddressField()
+    org_id = models.CharField(max_length=64, null=True)
+    user_agent = models.CharField(max_length=64)
```

Comment:
user agents can be pretty long, so lets increase the max length here

Reply:
I'd prefer to collect the useragent verbatim here. Also why not make it a `TextField`?